tool/model/comparison.py

```diff
@@ -85,47 +87,75 @@ def calculate_confidence(self):
         self.confidence = self.total / highest * 100
 
     def get_high_scoring_properties(self):
+        # Return scores that are over 0.5 - this is arbitrary, may want to increase
         return [k for k, v in self.scores.items() if v > 0.5]
 
     def compare_equals(self, first, second):
-        if first == second:
+        if first == second and first is not None:
```

Comment:
Does this mean that properties which have a `None` value don't count towards the score anymore?

Reply:
If one of them is missing, we don't need to bother comparing them. It does count towards the score though because any fields that are not compared at all because one or both is missing decreases overall confidence in the final calculation.